airflow/www/templates/airflow/graph.html

```diff
@@ -362,6 +362,15 @@
 
 </script>
 <script src="{{ url_for_asset('graph.js') }}"></script>
+<script src="{{ url_for_asset('index.js') }}"></script>
+<script type="text/javascript">
+  let dagTz = '{{ dag.timezone.name }}';
```

Comment:
We've started putting thins like this in `<meta>` tags, rather than blocks of JS (or we have open PRs by @mik-laj to do that. I forget if we've merged them yet)

Reply:
@ashb PR is still waiting for you. https://github.com/apache/airflow/pull/4787

Reply:
Cool, @mik-laj's PR looks good to me. I'll be happy to update this once #4787 is merged.